code/src/index.css

```diff
@@ -5,9 +5,182 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #000;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.movie-list {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.movie-list a{
+ position: relative;
+ text-decoration: none;
+ width: 25%;
+}
+
+.movie-list img{
+  width: 100%;
+  height: 100%;
+}
+
+.movie-info {
+  display: none;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 20px;
+  background: rgba(0,0,0,0.50);
+  color: whitesmoke;
+}
+
+.movie-list a:hover .movie-info{
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.back-link {
+  text-decoration: none;
+  color: whitesmoke;
+  font-weight: bold;
+  font-size: 20px;
+  padding-left: 40px;
+  display: flex;
+  align-items: center; 
+}
+
+.back-arrow {
+  width: 25px;
+  height: 25px;
+  margin-right: 5px;
+  transition: .4s ease-in-out;
+}
+
+.back-link:hover .back-arrow{
+  margin-right: 10px;
+}
+
+.movie-backdrop {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.movie-backdrop a {
+  padding-top: 70px;
+}
+
+.movie-details-container {
+  display: flex;
+  align-items: flex-end;
+  color: whitesmoke;
+  padding: 40px;
+}
+
+.movie-details {
+  padding-left: 15px;
+  width: 30%;
+}
+
+.movie-rating {
+  display: inline;
+  color: #f10400;
+  font-size: 17px;
+  margin-left: 20px;
+}
+
+.not-found-container {
+  display: flex;
+  align-items: center;
+  height: 500px;
+  justify-content: center;
+}
+
+.not-found-wrapper{
+  border: whitesmoke 1px solid;
+  border-radius: 10px;
+  padding: 50px;
+  text-align: center;
+}
+
+.not-found-container h2 {
+  padding-left: 30px;
+  color: whitesmoke;
+}
+
+.not-found-container a {
+  display: flex;
+  justify-content: center;
+}
+
+.loading-container {
+  display: flex;
+  justify-content: center;
+  height: 400px;
+  align-items: center;
+  background: white;
+}
+
+.loading-container h2 {
+  font-size: 50px;
+}
+
+@media (min-width: 667px) and (max-width: 1024px) {
+
+  .movie-list a {
+    width: 30%;
+    flex-grow: 1;
+  } 
+
+  .movie-backdrop{
+    position: relative;
+  }
+
+  .movie-details {
+    width: 90%;
+  }
+}
+
+@media (max-width: 666px) {
+
```

Comment:
If you start mobile first you probably wouldn't need this media query? Because then everything you do up until width 666px is "default"?